Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -660,6 +676,27 @@ def _invalidate(self, which=None):
 
         self.commit()
 
+    def show_column_chooser(self, pos):
+        # pylint doesn't know that self.shown_scores is a set, not a Setting
+        # pylint: disable=unsupported-membership-test
+        def update(col_name, state):
```

Comment:
Perhaps misleading since it would imply it is (was) already visible. I changed it to `checked`.